index.js

```diff
@@ -149,7 +155,15 @@ oauthPlugin.APPLE_CONFIGURATION = {
   authorizeHost: 'https://appleid.apple.com',
   authorizePath: '/auth/authorize',
   tokenHost: 'https://appleid.apple.com',
-  tokenPath: '/auth/token'
+  tokenPath: '/auth/token',
+  [kGenerateCallbackUriParams]: function (callbackUriParams, requestObject, scope, state) {
+    const stringifyScope = Array.isArray(scope) ? scope.join(' ') : scope
+    // `response_mode` must be `form_post` when scope include `email` or `name`
```

Comment:
could we link the apple documentation?

Reply:
no documentation for this, only error message tell you that you need this option to be `form_post`.

Reply:
The most related documentation is this one.
https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_js/incorporating_sign_in_with_apple_into_other_platforms